spec/memcached/client_spec.cr

```diff
@@ -129,4 +129,17 @@ describe Memcached::Client do
     client.decrement("dkey", 2, 0).should eq(3)
   end
 
+  it "fetches" do
```

Comment:
Could you please add a test case for the situation when a key is present in memcached? I think we should verify that we do not call set in this case.

Further, it's nice to test that we do not call set if passed block returns nil (see my other comment).